templates/firefox-ios.xliff

```diff
@@ -831,28 +850,72 @@
       </trans-unit>
-      <trans-unit id="Downloads.Alert.Message">
-        <source>Firefox is unable to download or display this file. Would you like to open it in another app?</source>
-        <note>The message of the alert box asking the user if they want to use another app to open a file.</note>
+      <trans-unit id="Downloads.CancelDialog.Message">
+        <source>Are you sure you want to cancel this download?</source>
+        <note>Alert dialog body when the user taps the cancel download icon.</note>
       </trans-unit>
-      <trans-unit id="Downloads.Alert.Title">
-        <source>Firefox Downloads</source>
-        <note>The title of the alert box asking the user if they want to use another app to open a file.</note>
+      <trans-unit id="Downloads.CancelDialog.Resume">
+        <source>Resume</source>
+        <note>Button declining the cancellation of the download.</note>
+      </trans-unit>
+      <trans-unit id="Downloads.CancelDialog.Title">
+        <source>Cancel Download</source>
+        <note>Alert dialog title when the user taps the cancel download icon.</note>
       </trans-unit>
       <trans-unit id="Downloads.Error.Message">
         <source>Downloads aren’t supported in Firefox yet.</source>
         <note>The message displayed to a user when they try and perform the download of an asset that Firefox cannot currently handle.</note>
       </trans-unit>
-      <trans-unit id="Enter your password to connect.">
-        <source>Enter your password to connect.</source>
+      <trans-unit id="Downloads.Toast.Cancelled.LabelText">
+        <source>Download Cancelled</source>
+        <note>The label text in the Download Cancelled toast for showing confirmation that the download was cancelled.</note>
+      </trans-unit>
+      <trans-unit id="Downloads.Toast.Failed.LabelText">
+        <source>Download Failed</source>
+        <note>The label text in the Download Failed toast for showing confirmation that the download has failed.</note>
+      </trans-unit>
+      <trans-unit id="Downloads.Toast.Failed.RetryButton">
+        <source>Retry</source>
+        <note>The button to retry a failed download from the Download Failed toast.</note>
+      </trans-unit>
+      <trans-unit id="Downloads.Toast.GoToDownloads.Button">
+        <source>Downloads</source>
+        <note>The button to open a new tab with the Downloads home panel</note>
+      </trans-unit>
+      <trans-unit id="Downloads.Toast.MultipleFiles.DescriptionText">
+        <source>1 of %d files</source>
+        <note>The description text in the Download progress toast for showing the number of files when multiple files are downloading.</note>
+      </trans-unit>
+      <trans-unit id="Downloads.Toast.MultipleFilesAndProgress.DescriptionText">
+        <source>%1$@ %2$@</source>
+        <note>The description text in the Download progress toast for showing the number of files (1$) and download progress (2$).</note>
```

Comment:
I wonder if we can be more clear here - at least, I'm getting confused (maybe it's just me). If we can link to a screenshot there, that might be helpful.

Reply:
@justindarc can you add a link to a screenshot or maybe the whole design to this one?

Reply:
@delphine does this help:

> The description text in the Download progress toast for showing the number of files (1$) and download progress (2$). This string only consists of two placeholders for purposes of displaying two other strings side-by-side where 1$ is Downloads.Toast.MultipleFiles.DescriptionText and 2$ is Downloads.Toast.Progress.DescriptionText. This string should only consist of the two placeholders side-by-side separated by a single space and 1$ should come before 2$ everywhere except for right-to-left locales.

Reply:
Yes, that helps clarify things, thanks